packages/components/popover/src/PopoverClose.tsx

```diff
@@ -0,0 +1,42 @@
+import * as RadixPopover from '@radix-ui/react-popover'
+import { Icon } from '@spark-ui/icon'
+import { IconButton } from '@spark-ui/icon-button'
+import { Close as CloseSVG } from '@spark-ui/icons/dist/icons/Close'
+import { cx } from 'class-variance-authority'
+import { forwardRef, useEffect } from 'react'
+
+import { usePopover } from './PopoverContext'
+
+export type CloseProps = RadixPopover.PopoverCloseProps & {
+  'aria-label': string
+}
+
+export const Close = forwardRef<HTMLButtonElement, CloseProps>(
+  ({ 'aria-label': ariaLabel, className, ...rest }, ref) => {
+    const { setHasCloseButton } = usePopover()
+
+    useEffect(() => {
+      setHasCloseButton(true)
+
+      return () => setHasCloseButton(false)
+    })
+
+    return (
+      <RadixPopover.Close
+        data-spark-component="Popover.Close"
+        ref={ref}
+        className={cx('absolute top-md right-md', className)}
+        asChild
+        {...rest}
+      >
+        <IconButton size="sm" intent="neutral" design="ghost" aria-label={ariaLabel}>
```

Comment:
- I think that `aria-label` could be part of `rest`
- Why not making `size`, `intent` and `design` configurable ? and with those values by default ?

Reply:
It is not part of the specs. And making the size configurable would be extremely tricky as the padding-right of `Popover.Content` must be adjusted when the `Popover.Close` is rendered to avoid overlapping.